src/Platforms/DB2Platform.php

```diff
@@ -241,20 +244,50 @@ public function getIndexDeclarationSQL(Index $index): string
      */
     protected function _getCreateTableSQL(string $name, array $columns, array $options = []): array
     {
-        $indexes = [];
-        if (isset($options['indexes'])) {
-            $indexes = $options['indexes'];
+        $columnListSql = $this->getColumnDeclarationListSQL($columns);
+
+        if (isset($options['uniqueConstraints']) && ! empty($options['uniqueConstraints'])) {
+            foreach ($options['uniqueConstraints'] as $definition) {
+                $columnListSql .= ', ' . $this->getUniqueConstraintDeclarationSQL($definition);
+            }
         }
 
-        $options['indexes'] = [];
+        if (isset($options['primary']) && ! empty($options['primary'])) {
+            $columnListSql .= ', PRIMARY KEY(' . implode(', ', array_unique(array_values($options['primary']))) . ')';
+        }
+
+        $statement = match ($options['temporary'] ?? '') {
+            '' => 'CREATE TABLE ',
+            'created' => 'CREATE GLOBAL TEMPORARY TABLE ',
+            'declared' => 'DECLARE GLOBAL TEMPORARY TABLE ',
+            default => throw new InvalidArgumentException(sprintf(
+                'invalid temporary specification for table %s',
+                $name,
+            ))
+        };
```

Comment:
Okay, so for MySQL we demand that the `temporary` property is a bool, but for DB2 we want it to be a string? Can you elaborate on the difference between those two options? Do we need to support both?